packages/main/src/Slider.ts

```diff
@@ -264,6 +264,14 @@ class Slider extends SliderBase implements IFormInputElement {
 		this._valueOnInteractionStart = undefined;
 	}
 
+	_onkeyup() {
+		this._onKeyupBase();
+
+		if (this._valueOnInteractionStart !== this.value) {
```

Comment:
In this way change event is fired on every **focusin** with [Tab] since **this._valueOnInteractionStart** is **undefined** and **this.value** is **0**, which is **true**.